server/src/api/feature/controller.ts

```diff
@@ -1,26 +1,47 @@
+import { LoginError } from '../users/controller'
 import FeatureDAO, { Feature } from './model'
 
 export default class FeatureController {
   constructor(private dao: FeatureDAO) {}
 
-  public async createFeature(feature: Feature): Promise<number> {
-    const featureId = await this.dao.createFeature(feature)
+  public async createFeature(feature: Feature, token: string): Promise<number> {
+    const userId = await this.dao.getUserIdFromSession(token)
+    if (userId === undefined) {
```

Comment:
I feel like there should be a validation layer above this so we don't have to do this check in every handler.